airflow/www/static/js/main.js

```diff
@@ -286,4 +286,8 @@ $(document).ready(() => {
 
   // Global Tooltip selector
   $(".js-tooltip").tooltip();
+
+  // Issue 44019: Turn off autocomplete for login form
```

Comment:
We usually don't link to issues in comments like this unless it is a todo comment.

Reply:
Okay, no worries. I have removed the issue reference.